{{cookiecutter.project_slug}}/package.json

```diff
@@ -1,38 +0,0 @@
-{
-  "name": "{{cookiecutter.project_slug}}",
-  "version": "{{ cookiecutter.version }}",
-  "dependencies": {},
-  "devDependencies": {
-    {% if cookiecutter.js_task_runner == 'Gulp' -%}
-    {% if cookiecutter.custom_bootstrap_compilation == 'y' -%}
-    "bootstrap": "4.3.1",
```

Comment:
The `frontend` project uses Bootstrap 5:

https://github.com/AccordBox/python-webpack-boilerplate/blob/e0ca53cab5301512e4bdd19cc3b217ec05fa8723/frontend_template/%7B%7Bcookiecutter.project_slug%7D%7D/package.json#L55

That's probably going to break things since Crispy forms is configured to use Bootstrap 4:

https://github.com/pydanny/cookiecutter-django/blob/ebc2f0446ac1cd22f5817f8bb63b0bda1d22ea51/%7B%7Bcookiecutter.project_slug%7D%7D/config/settings/base.py#L210-L211

Reply:
Regarding the concern about breaking changes between an external package, we can add `checkout` argument for cookiecutter to choose a specific branch, tags, or commit.

https://cookiecutter.readthedocs.io/en/1.7.2/usage.html#works-directly-with-git-and-hg-mercurial-repos-too